src/common/product_models/columnar_attribs_common_models.py

```diff
@@ -66,6 +79,18 @@ class AttributesColumnSpec(BaseModel):
         description="The filter strategy for the column if any. Not all column types need "
             + "a filter strategy."
     )] = None
+    display_name: Annotated[str | None, Field(
```

Comment:
I think that display name and category should be required, right?

Reply:
there are genome attribute columns such as `_mtchsel`, `load_ver` and `coll` doesn't have display name and category. 
We also convert heatmap column to genome attrib col for heatmap filtering which doesn't require display_name and category fields.

Reply:
> _mtchsel, load_ver and coll doesn't have display name 

We could just put in arbitrary placeholders for those

> We also convert heatmap column to genome attrib col for heatmap filtering which doesn't require display_name and category fields.

Hmm, this is pointing out a code smell since we're making fields that should be required optional because of a filtering implementation detail. We should probably fix this long term, but what I would do for now is just add arbitrary placeholder text as a hack when making the conversion and add a TODO that we should think through how this works later and fix the hack

Reply:
maybe something like a non-visible bool flag that's default false. If the flag is set then display name and column are expected to be missing

Reply:
Actually maybe that's the way to go right now, and we could also update the genome meta endpoint to automatically remove non-visible columns

Reply:
👍

Reply:
Can you update these lines to key off of column visibilty https://github.com/kbase/collections/blob/main/src/service/data_products/genome_attributes.py#L272-L273

Reply:
👍

Reply:
I think COLLECTION_KEYS is an unused import at this point

Reply:
👍 removed.